packages/flutter/lib/src/widgets/system_context_menu.dart

```diff
@@ -83,7 +83,7 @@ class SystemContextMenu extends StatefulWidget {
         ),
       ),
       items: items ?? getDefaultItems(editableTextState),
-      onSystemHide: editableTextState.hideToolbar,
+      onSystemHide: () => editableTextState.hideToolbar(false),
```

Comment:
Hm in hindsight it'd be nice if this was a named argument instead of a positional argument. It's hard to tell that boolean sets the handle visibility.

Reply:
Definitely agree with this. I kind of wish the hideToolbar method didn't do this at all and that there was a separate method to hide the handles, or that the method had a more appropriate name. SelectionOverlay and TextSelectionOverlay expose a hideHandles but we don't expose it at the editable text level.